quodlibet/browsers/covergrid/main.py

```diff
@@ -119,7 +119,8 @@ class CoverGrid(Browser, util.InstanceTracker, VisibleUpdate,
     __last_render = None
     __last_render_surface = None
 
-    _PATTERN_FN = os.path.join(quodlibet.get_user_dir(), "album_pattern")
+    # TODO: Maybe on the Config folder?
+    _PATTERN_FN = os.path.join(quodlibet.get_data_dir(), "album_pattern")
```

Comment:
Not sure this is data or configuration.

Reply:
definitely the same as the above either way - so I think config